Mathlib/LinearAlgebra/SymmetricAlgebra/Basic.lean

```diff
@@ -139,3 +144,72 @@ instance [Nontrivial R] : Nontrivial (SymmetricAlgebra R M) :=
   (algebraMap_leftInverse M).injective.nontrivial
 
 end SymmetricAlgebra
+
+variable {A : Type*} [CommSemiring A] [Algebra R A] (f : M →ₗ[R] A)
+variable {R} {M}
+
+/-- Given a morphism `ι : M →ₗ[R] A`, `IsSymmetricAlgebra ι` is a proposition saying that the
+algebra homomorphism from `SymmetricAlgebra R M` to `A` is bijective. -/
+def IsSymmetricAlgebra (f : M →ₗ[R] A) : Prop :=
+  Function.Bijective (SymmetricAlgebra.lift f)
```

Comment:
I suspect it ends up more convenient to use
```suggestion
structure IsSymmetricAlgebra (f : M →ₗ[R] A) where
  lift : A →ₐ[R] SymmetricAlgebra R M
  comp_lift : (SymmetricAlgebra.lift f).comp lift = .id _ _
  lift_comp : lift.comp (SymmetricAlgebra.lift f) = .id _ _
```
such that `equiv` is computable and has nice defeqs.

Reply:
(feel free to wait for other reviewer opinions before trying to change it)